docs/run/prepare/test-command.md

```diff
@@ -8,15 +8,73 @@ import TabItem from '@theme/TabItem';
+### Pre-requisites
+
+#### Running after you have started cluster creation
+
+All prerequisites for each category apply to `test all`.
+
+### Run
+
+```shell
+./charon test all \
+  --beacon-endpoints="https://ethereum-holesky-beacon-api.publicnode.com,https://ethereum-sepolia-beacon-api.publicnode.com" \
+  --mev-endpoints="https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net" \
+  --peers-enrs="enr:-HW4QNDXi9MzdH9Af65g20jDfelAJ0kJhclitkYYgFziYHXhRFF6JyB_CnVnimB7VxKBGBSkHbmy-Tu8BJq8JQkfptiAgmlkgnY0iXNlY3AyNTZrMaEDBVt5pk6x0A2fjth25pjLOEE9DpqCG-BCYyvutY04TZ,enr:-HW4QO2vefLueTBEUGly5hkcpL7NWdMKWx7Nuy9f7z6XZInCbFAc0IZj6bsnmj-Wi4ElS6jNa0Mge5Rkc2WGTVemas2AgmlkgnY0iXNlY3AyNTZrMaECR9SmYQ_1HRgJmNxvh_ER2Sxx78HgKKgKaOkCROYwaDY" \
+  --output-json="/opt/charon/test/test-result.json"
+```
+
+Load test
+
+```shell
+./charon test all \
+  --beacon-endpoints="https://ethereum-holesky-beacon-api.publicnode.com,https://ethereum-sepolia-beacon-api.publicnode.com" \
+  --beacon-load-test \
+  --mev-endpoints="https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net" \
+  --mev-beacon-node-endpoint="https://ethereum-beacon-api.publicnode.com" \
+  --mev-load-test \
+  --peers-enrs="enr:-HW4QNDXi9MzdH9Af65g20jDfelAJ0kJhclitkYYgFziYHXhRFF6JyB_CnVnimB7VxKBGBSkHbmy-Tu8BJq8JQkfptiAgmlkgnY0iXNlY3AyNTZrMaEDBVt5pk6x0A2fjth25pjLOEE9DpqCG-BCYyvutY04TZ,enr:-HW4QO2vefLueTBEUGly5hkcpL7NWdMKWx7Nuy9f7z6XZInCbFAc0IZj6bsnmj-Wi4ElS6jNa0Mge5Rkc2WGTVemas2AgmlkgnY0iXNlY3AyNTZrMaECR9SmYQ_1HRgJmNxvh_ER2Sxx78HgKKgKaOkCROYwaDY" \
+  --output-json="/opt/charon/test/test-result.json"
+```
+
+### Run with Docker
+
+```shell
+docker run -v "$(pwd):/opt/charon/test" obolnetwork/charon:v1.1.2 test all \
+  --beacon-endpoints="https://ethereum-holesky-beacon-api.publicnode.com,https://ethereum-sepolia-beacon-api.publicnode.com" \
+  --beacon-simulation-file-dir="/opt/charon/test" \
+  --mev-endpoints="https://0xa7ab7a996c8584251c8f925da3170bdfd6ebc75d50f5ddc4050a6fdc77f2a3b5fce2cc750d0865e05d7228af97d69561@agnostic-relay.net,https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net,global.titanrelay.xyz" \
+  --peers-enrs="enr:-HW4QBHlcyD3fYWUMADiOv4OxODaL5wJG0a7P7d_ltu4VZe1MibZ1N-twFaoaq0BoCtXcY71etxLJGeEZT5p3XCO6GOAgmlkgnY0iXNlY3AyNTZrMaEDI2HRUlVBag__njkOWEEQRLlC9ylIVCrIXOuNBSlrx6o,enr:-HW4QDwUF804f4WhUjwcp4JJ-PrRH0glQZv8s2cVHlBRPJ3SYcYO-dvJGsKhztffrski5eujJkl8oAc983MZy6-PqF2AgmlkgnY0iXNlY3AyNTZrMaECPEPryjkmUBnQFyjmMw9rl7DVtKL0243nN5iepqsvKDw" \
+  --infra-disk-io-test-file-dir="/opt/charon/test" \
+  --data-dir="/opt/charon/test/.charon" \
```

Comment:
It won't store anything. It will use the enr for the peers test. I used the already existing `--data-dir` flag. Probably we need to change its description.

Reply:
If it doesn't store anything, maybe we should be like `charon run`, where we moved away from `data-dir` and instead had `--private-key-file` ? (looks like we forgot to do the same in `charon dkg`...) Or at least the description change maybe, but think an exact flag for the enr might the most precise and inline with the rest of our commands. 

https://docs.obol.org/next/learn/charon/charon-cli-reference#the-run-command

Reply:
Good point, will update that now!